.swcrc

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://swc.rs/schema.json",
+    "jsc": {
+        "parser": {
+            "syntax": "typescript",
+            "jsx": true
+        },
+        "transform": {
+            "react": {
+                "runtime": "automatic"
+            }
+        },
+        "externalHelpers": true,
+        "experimental": {}
+    }
+}
```

Comment:
@jandrade  Circling back to this PR as something came up in Perseus. SWC supports polyfilling for older browsers, but you have to configure it. We're going to be enabling the [browser support](https://swc.rs/docs/configuration/supported-browsers) in Perseus shortly, but you might need to do that here also for WB. We had a regression slip into production where the use of `Array.at` isn't supported in Safari 15.